packages/abi-typegen/src/runTypegen.test.ts

```diff
@@ -189,7 +189,7 @@ describe('runTypegen.js', () => {
     // validates execution was ok
     expect(error).toBeFalsy();
 
-    expect(stdoutWrite).toHaveBeenCalledTimes(4);
+    expect(stdoutWrite).toHaveBeenCalledTimes(5);
```

Comment:
Increased to 5 because now the `./common.d.ts` file gets generated as well and that's logged in the console.